docs/usage/configuration-options.md

```diff
@@ -676,6 +676,9 @@ Example:
 
 ## gitLabAutomerge
 
+Attention: This is a top-level key.
+To activate automerging within a `packageRule`, you must set `automerge` to `true` in that same `packagaRule`.
```

Comment:
```suggestion
To activate automerging within a `packageRule`, you must set `automerge` to `true` in the same `packageRule`.
```

Whoops, made a spelling error.

And maybe `in the same` is better than `in that same`, but not sure about that one.

Reply:
Automerge can be also globally used. Or you can use one of our automerge presets.

So this docs should describe that this gitlab flag only sets the gitlab flag when automerge is used too.

Reply:
> 
> 
> Automerge can be also globally used. Or you can use one of our automerge presets.
> 
> So this docs should describe that this gitlab flag only sets the gitlab flag when automerge is used too.

How would you communicate this?

Reply:
🤔

Maybe something like:

If you are using automerge you can use this setting to use the gitlab native automerge function for MR's to speedup automerge.

Reply:
I've updated the file